CHANGELOG.md

```diff
@@ -1,17 +1,17 @@
-# Release 0.16.0-dev
-
-### New features
-
-### Improvements
-
-### Breaking changes
+# Release 0.15.1
 
 ### Bug fixes
 
+* Fixes a bug with numpy where building the wheels with numpy version >1.20
+  would cause installation issues.
```

Comment:
```suggestion
* The PennyLane-Lightning binaries are now built with NumPy 1.19.5, to avoid ABI
  compatibility issues with the latest NumPy 1.20 release. See
  [the NumPy release notes](https://numpy.org/doc/stable/release/1.20.0-notes.html#size-of-np-ndarray-and-np-void-changed)
  for more details.
```